docs/config-app.md

```diff
@@ -23,6 +23,7 @@ This parameter affects how many CPU cores will be utilized by the application. R
 - `http-client.connect-timeout-ms` - set the connect timeout.
 
 ## Auction
+- `auction.update-rates-period` - default default period to update currency rates.
```

Comment:
"default" has been duplicated. Additionally it could be placed alongside `ad-server-targeting` parameter